bundles/statistics/statsgrid2016/Tile.js

```diff
@@ -63,22 +63,32 @@ function(instance, service) {
      */
     startPlugin : function() {
         this._addTileStyleClasses();
+    },
+    extend: function () {
```

Comment:
I would rename this to something like setupTools and pass a reference to the flyoutmanager instead of fetching it through instance.